validation/standard/standard-versions.toml

```diff
@@ -0,0 +1,68 @@
+["op-contracts/v1.4.0-rc.3"]
+optimism_mintable_erc20_factory = "1.9.0"
+optimism_portal = "3.8.0"
+system_config = "2.2.0"
+l1_erc721_bridge = "2.1.0"
+l1_standard_bridge = "2.1.0"
+address_manager = ""
+l1_cross_domain_messenger = "2.3.0"
+protocol_versions = "1.0.0"
```

Comment:
Let's keep this contract in the TOML file, but we can't check its version at the moment, see https://github.com/ethereum-optimism/superchain-registry/pull/279.